osidb/models.py

```diff
@@ -853,6 +855,7 @@ class Meta:
         )
         indexes = TrackingMixin.Meta.indexes + [
             models.Index(fields=["-cve_id"]),
+            models.Index(fields=["-local_updated_dt"]),
```

Comment:
As prev mentioned ... you might find adding indexes for existing **updated_dt** a better first approach (Flaw, Affect and Tracker).